lms/djangoapps/certificates/tasks.py

```diff
@@ -38,7 +38,7 @@ def generate_certificate(self, **kwargs):
         actual_verification_status = IDVerificationService.user_status(student)
         actual_verification_status = actual_verification_status['status']
         if expected_verification_status != actual_verification_status:
-            logger.warn(u'Expected verification status {expected} '
```

Comment:
Since it's a warning message use `logger.warning` instead of `info`.

Reply:
Updated `logger.warn` to `logger.warning`.